.travis.yml

```diff
@@ -37,6 +37,10 @@ env:
                    -DDEPS_PREFIX=$DEPS_BUILD_DIR/usr
                    -DMIN_LOG_LEVEL=3"
     - DEPS_CMAKE_FLAGS="-DUSE_BUNDLED_GPERF=OFF"
+    # Additional CMake flags for 32-bit builds.
+    - CMAKE_FLAGS_32BIT="-DCMAKE_SYSTEM_LIBRARY_PATH=/lib32:/usr/lib32:/usr/local/lib32
+                         -DCMAKE_IGNORE_PATH=/lib:/usr/lib:/usr/local/lib
+                         -DCMAKE_TOOLCHAIN_FILE=$TRAVIS_BUILD_DIR/cmake/i386-linux-gnu.toolchain.cmake"
```

Comment:
why is this needed?

Reply:
I don't know what the cause is, but in the CI environment 64bit `libacl` was used by linking and the build failed, so I needed to set it.

Reply:
This should be needed. We used to have something like this, but removed it recently: 3d3c783fead3dc328ede3ccd0fb7b3befcad191a

maybe @blueyed has insight